client/extensions/relatedArticles/src/EventsRelatedArticles/EventsRelatedArticlesModal.tsx

```diff
@@ -0,0 +1,307 @@
+import React from 'react';
+import {
+    SearchBar,
+    Modal,
+    Dropdown,
+    Spacer,
+    Button,
+    WithPagination,
+    Loader,
+    Panel,
+    PanelHeader,
+    PanelContent,
+    PanelContentBlock,
+    ContentDivider,
+} from 'superdesk-ui-framework/react';
+import {getProjectedFieldsArticle, gettext} from 'superdesk-core/scripts/core/utils';
```

Comment:
Please do not import anything from `superdesk-core`, it was something that we used to do years ago before the SuperdeskAPI. Now that we have that API it is best to use that, if needing anything from core then we should add the required functionality to that API.